app/src/androidTest/java/com/mapbox/maps/testapp/observable/ObservableExtensionTest.kt

```diff
@@ -142,11 +149,15 @@ class ObservableExtensionTest : BaseMapTest() {
     if (!latch.await(20000, TimeUnit.MILLISECONDS)) {
       throw TimeoutException()
     }
-    mapboxMap.removeOnMapLoadErrorListener(listener)
+
+    rule.scenario.onActivity { activity ->
+      activity.runOnUiThread {
+        mapboxMap.removeOnMapLoadErrorListener(listener)
```

Comment:
just curious - don't we see any errors / warning in logcat that we're trying to access outdated map object when running this test?